client/src/components/content/BpmnModeler/BpmnModeler.jsx

```diff
@@ -75,9 +74,13 @@ const BpmnModeler = () => {
           <div className={styles['bpmn-modeler-container']} id='bpmnview' />
         </div>
       </Content>
-      <ModelerSidebar modeler={modeler} getRobotFile={downloadRobotFile} />
+      <ModelerSidebar robotId={robotId} modeler={modeler} getRobotFile={downloadRobotFile} />
     </Layout>
   );
 };
 
 export default BpmnModeler;
+
+BpmnModeler.propTypes = {
+  robotId: PropTypes.string.isRequired
+};
```

Comment:
That looks cool, but I dont understand it! Can you explain it or maybe link me to some resource I can read through?

Reply:
https://reactjs.org/docs/typechecking-with-proptypes.html

Reply:
Thanks Daniel! (@SanJSp I've implemented it in every frontend-function where necessary to satisfy the linter)